src/main/resources/reference.conf

```diff
@@ -1290,20 +1290,45 @@ resourceTypes = {
       "unlock_resource" = {
         description = "Can unlock a resource"
       }
+      "read_aggregate_data" = {
+        description = "Can read aggregate data about a snapshot"
+      }
+      "create_snapshot_request" = {
+        description = "Can create a snapshot request for this snapshot"
+      }
+      "get_snapshot_builder_settings" = {
+        description = "Can get the snapshot builder settings for this snapshot"
+      }
+      "update_snapshot_builder_settings" = {
+        description = "Can get the snapshot builder settings for this snapshot"
+      }
     }
     ownerRoleName = "steward"
     roles = {
       steward = {
-        roleActions = ["delete", "edit_datasnapshot", "update_snapshot", "read_data",  "discover_data",  "share_policy::steward", "share_policy::custodian", "share_policy::reader",  "share_policy::discoverer", "read_policies", "set_public", "update_passport_identifier", "export_snapshot", "view_journal", "read_auth_domain", "update_auth_domain", "lock_resource", "unlock_resource"]
+        roleActions = ["share_policy::steward", "share_policy::custodian", "update_passport_identifier", "view_journal"]
```

Comment:
Here is where I really see the improved clarity: stewards and custodians can both do quite a bit, and it's been hard to grasp what elevated permissions a steward would have over a custodian!